src/widgetastic/browser.py

```diff
@@ -156,6 +156,13 @@ def url(self, address):
         self.logger.info('Opening URL: %r', address)
         self.selenium.get(address)
 
+    @property
+    def title(self):
+        """Returns current title"""
+        result = self.selenium.title
+        self.logger.info('current title -> %r', result)
```

Comment:
Please use more consistent log formatting here, note the logging for the `url` properties.  Should be `'Current title: %r'`

Reply:
done